src/components/footer/footer.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Block } from 'baseui/block';
+import { Footer } from '../../types/homepage';
+import {
+  ALIGN,
+  StyledNavigationItem,
+  StyledNavigationList,
+} from 'baseui/header-navigation';
+
+declare type FooterSectionProps = {
+  data: Footer;
+};
+
+export const FooterSection: React.FC<FooterSectionProps> = ({ data }) => {
+  return (
+    <Block backgroundColor={'#f5f5f5'} padding={['scale700']}>
```

Comment:
I have mentioned this couple of times, hopefully this is the last time that this feedback needs to be given - Please avoid using colors like this color - font color, size, spacing etc should come from baseweb theme classes.